23-struct.md

```diff
@@ -392,7 +392,8 @@ type People1 struct {
 	name string
 	age  int
 }
-type People2 = struct {
+
+type People2 struct {
```

Comment:
this is not type alias. this is just new type declaration. example of type alias declaration:

```
type A = B
type A = struct {
    field string
}
```

more information https://golang.org/doc/go1.9#language